index.html

```diff
@@ -104,15 +104,15 @@ <h3 class="article__title">How can I help?</h3>
             </div>
           </div>
         </div>
-
-        <div class="sidebar">
+ 
+        <aside class="sidebar">
           <h2 class="heading-underline">Upcoming events</h2>
 
           <div class="article">
             <img
```

Comment:
The image and the text are not aligned as per the homework's screenshot (i.e. image on the left of the text). Would you be able to do that?